Bottom: 4770270a81803d669a2efd3cf042fdfb55a23bd4
Top:    b84aef9032037a18ba053e9b409a9437c4173adb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 00:59:00 +0100

action.c: Remove commented out line of code

Left from commit 089fbaf59c78fe75475db737e7e2827cd745d570
(Initial C++ compatibility patch)


---

diff --git a/src/action.c b/src/action.c
index ea9a51f..22e78bd 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2758,7 +2758,6 @@ ActionDisplay (int argc, char **argv, int childX, int childY)
 	  break;
 
 	case F_CycleCrosshair:
-	  //Crosshair.shape++;
 	  Crosshair.shape = CrosshairShapeIncrement(Crosshair.shape);
 	  if (Crosshair_Shapes_Number == Crosshair.shape)
 	    Crosshair.shape = Basic_Crosshair_Shape;
